src/Diagnostics.DataProviders/IWawsObserverTokenService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Diagnostics.DataProviders
+{
+    public interface IWawsObserverTokenService
```

Comment:
Since this is defining the same interface as ISupportBayApiObserverTokenService, could we simplify to one interface or are they likely to diverge?

Reply:
So I did this because of the dependency injection in the [StartUp.cs](https://github.com/Azure/Azure-AppServices-Diagnostics/pull/141/files#diff-d567401b55ead5c2a4a3f4a20ed0fc7c). I have to specify some kind of type hence the two interfaces to delineate between two different token services that can be injected and used. See the [DiagnosticRequestMiddleware.cs](https://github.com/Azure/Azure-AppServices-Diagnostics/pull/141/files#diff-058205c71f6a0cc2f65ad7406829861a) on how I'm using it. If I had just specified `ObserverTokenService` as my service type when using `services.AddSingleton` in the StartUp.cs file then it will make it ambigious